src/lib/statsv2.svelte

```diff
@@ -0,0 +1,179 @@
+<!--
+  statsv2.svelte:
+    Stats highlighting the value of fixNation's work-v2
+-->
+<script lang="ts">
+  import { base } from '$app/paths';
+  import { onMount, onDestroy } from 'svelte';
+  let sections = [
+    {
+      number: '1',
+      subtitle: 'largest clinic for homeless cats',
+      picture: 'statsv2/statImage1.png'
+    },
+    {
+      number: '115k',
+      subtitle: 'total cats spayed or neutered',
+      picture: 'statsv2/statImage2.png'
+    },
+    {
+      number: '12k',
+      subtitle: 'TNR volunteers trained',
+      picture: 'statsv2/statImage3.png'
+    },
+    {
+      number: '75%',
+      subtitle: 'cats are feral',
+      picture: 'statsv2/statImage4.png'
+    },
+    {
+      number: '89%',
+      subtitle: 'profits go directly to FixNation programs',
+      picture: 'statsv2/statImage5.png'
+    },
+    {
+      number: '21%',
+      subtitle: 'of cats abandoned because their of their residency',
+      picture: 'statsv2/statImage6.png'
+    }
+  ];
+
+  let screenWidth: number;
+
+  const updateScreenWidth = () => {
+    screenWidth = window.innerWidth;
+  };
+
+  onMount(() => {
+    if (typeof window !== 'undefined') {
+      window.addEventListener('resize', updateScreenWidth);
+      updateScreenWidth();
+    }
+  });
+
+  onDestroy(() => {
+    if (typeof window !== 'undefined') {
+      window.removeEventListener('resize', updateScreenWidth);
+    }
+  });
+</script>
```

Comment:
You can get the screen width more concisely by using a Svelte bind:

```suggestion
</script>

<svelte:window bind:innerWidth={screenWidth} />
```